templates/view.html

```diff
@@ -61,49 +60,71 @@ <h3 style="word-break:break-all" {{if eq .Status 2}}class="remake" {{end}} {{if
                 <hr>
                 {{end}}
                 <div class="row">
-                <div class="col-md-4">{{T "seeders"}}</div>
-                <div class="col-md-4">{{T "leechers"}}</div>
-                <div class="col-md-4">{{T "completed"}}</div>
+                    <div class="col-md-4">{{T "seeders"}}</div>
+                    <div class="col-md-4">{{T "leechers"}}</div>
+                    <div class="col-md-4">{{T "completed"}}</div>
                 </div>
                 <div class="row">
-                <div class="col-md-4">{{.Seeders}}</div>
-                <div class="col-md-4">{{.Leechers}}</div>
-                <div class="col-md-4"><span class="completed" style="">{{.Completed}}</span></div>
-                <div class="col-md-8">
-                <div class="proSeedBar">
-                  <div class="seeds" style="width: {{ calcWidthSeed .Seeders .Leechers }}%;"></div>
-                  <div class="leechs" style="width: {{ calcWidthLeech .Seeders .Leechers }}%;"></div>
-                </div>
-                </div>
+                    <div class="col-md-4">{{.Seeders}}</div>
+                    <div class="col-md-4">{{.Leechers}}</div>
+                    <div class="col-md-4"><span class="completed" style="">{{.Completed}}</span></div>
+                    <div class="col-md-8">
+                        <div class="proSeedBar">
+                            <div class="seeds" style="width: {{ calcWidthSeed .Seeders .Leechers }}%;"></div>
+                            <div class="leechs" style="width: {{ calcWidthLeech .Seeders .Leechers }}%;"></div>
+                        </div>
+                    </div>
                 </div>
                 <hr>
             </div>
         </div>
         <div class="row" id="comments">
             <div class="col-md-12">
-                <h4>{{T "comments"}}</h4>
-                {{ range $index, $element := .Comments }}
-                <div class="row comment-row" id="comment_{{$index}}">
-                    <div class="col-md-2">
-                        {{if eq .UserID -1}}
-                            <a href="#comment_{{$index}}">{{$index}}</a>
-                            {{.Username}}
-                        {{else}}
-                            <a href="#comment_{{$index}}">{{$index}}</a>
-                            <a href="{{$.URL.Parse (printf "/user/%d/-" .UserID ) }}">{{.Username}}</a>
+                <div class="commentList">
+                    <h4>{{T "comments"}}</h4>
+                    <hr/>
+                    <ul class="comments">
+                        {{ range $index, $element := .Comments }}
+                        <li class="clearfix comment" id="comment_{{$index}}">
+                            {{/* The following line is wrong. Should be using the MD5 User hash but i don't knwo how to access it. */}}
+                            <img src="https://www.gravatar.com/avatar/{{ .UserID }}?s=50" class="avatar" alt="">
```

Comment:
@akuma06 Using {{ .MD5 }} thows the following error:

`template: view.html:90:73: executing "content" at <.MD5>: can't evaluate field MD5 in type model.CommentJSON`